mobile-save-for-later/src/main/scala/com/gu/sfl/local/LocalServer.scala

```diff
@@ -0,0 +1,35 @@
+package com.gu.sfl.local
+
+import cats.data.Kleisli
+import cats.effect.{ExitCode, IO, IOApp}
+import com.gu.sfl.lambda.{FetchArticlesLambda, LambdaRequest, LambdaResponse, SaveArticlesLambda}
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.dsl.io._
+import org.http4s.implicits.http4sKleisliResponseSyntaxOptionT
+import org.http4s.server.Router
+import org.http4s.{HttpRoutes, Request, Response}
+
+object LocalServer extends IOApp {
+
+  val route: HttpRoutes[IO] = HttpRoutes.of[IO] {
+    case request @ GET -> Root / "syncedPrefs" / "me" =>
+      val lambsResFt = FetchArticlesLambda.fetchArticlesController.apply(LambdaRequest.fromHttp4sRequest(request))
+      IO.fromFuture(IO(lambsResFt)).map(LambdaResponse.toHttp4sRes)
+
+    case request @ POST -> Root / "syncedPrefs" / "me" / "savedArticles" =>
+      val lambsResFt = SaveArticlesLambda.saveArticlesController.apply(LambdaRequest.fromHttp4sRequest(request))
+      IO.fromFuture(IO(lambsResFt)).map(LambdaResponse.toHttp4sRes)
+  }
+
+  val httpApp: Kleisli[IO, Request[IO], Response[IO]] = Router(
+    "/" -> route
+  ).orNotFound
+
+  import scala.concurrent.ExecutionContext.global
+  override def run(args: List[String]) =
+    BlazeServerBuilder[IO](global)
+      .bindHttp(8080, "localhost")
```

Comment:
For some reason, port 8080 is claimed by `nginx` on my machine (`nginx` is quite commonly used by projects at the Guardian, facilitated by [`guardian/dev-nginx`](https://github.com/guardian/dev-nginx))

I don't know if it's worth switching to a less-commonly-used port to avoid that sort of thing?!

Reply:
yeah good idea